<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionsManagement</class>
 <widget class="QDialog" name="SessionsManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session management</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../risorse.qrc">
    <normaloff>:/icon/images/icon.png</normaloff>:/icon/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Session list</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="sessionList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="activateCmd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newCmd">
         <property name="text">
          <string>New...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editCmd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteCmd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedName">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedDescription">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>----</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SessionDetailWidget" name="details" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadFileCmd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportFilesCmd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export file data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>848</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SessionDetailWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/sessiondetailwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../risorse.qrc"/>
  <include location="../../risorse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SessionsManagement</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SessionsManagement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
